api/base.proto

```diff
@@ -64,3 +65,18 @@ enum RoutingPriority {
   DEFAULT = 0;
   HIGH = 1;
 }
+
+// Header name/value pair.
+message HeaderValue {
+  // Header name.
+  string key = 1;
+
+  // Header value. The same format specifier as used for HTTP access logging
+  // applies here, however unknown header values are replaced with the empty
+  // string instead of -. [V2-API-DIFF].
+  string value = 2;
+
+  // Should the value be appended? If false (default), the value overrides
+  // existing values [V2-API-DIFF].
+  google.protobuf.BoolValue append = 3;
```

Comment:
Eek, I forgot I made that change recently. Also, the above one about -/empty was also a recent one. Might be best not share the definition in light of this.